scripts/examples/gn_asr_example.sh

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
```

Comment:
please integrate with scripts/build_examples.py instead of providing more custom shell scripts